guides/SPI.md

```diff
@@ -21,6 +21,21 @@ In order to become a _service provider_, you have two options:
 * Register an application in the Wix Developer Center, and add the SPI component that you wish to implement. 
 * Become a _service provider_ in a single site by implementing the SPI using Velo. 
 
+## Integration with Wix
+In order to enable Wix applications to communicate with your provider application, Wix needs to know some configuration details. You add the configurations via the Dev Center by entering your app’s url and method and other configuration details in the **JSON Editor** in the service’s extension description page. 
+The **Instructions** section of the extension description page provides a list of the required configuration parameters and types. 
+
+![alt text](../media/spi-config.png)
+
+
+In order to integrate the app you registered in the Wix Developer Center, provide configuration details in the **JSON Editor**:
+1. Click on **Components** in the Dashboard sidebar.
```

Comment:
Missing some sort of a navigation prompt. In other words, how do I get here?

I'm not sure it's called the Dashboard sidebar (**Dashboard** is an option in the menu, but not the name of the menu itself). Check with the team to see if anyone knows what the proper term is. If you can't find a term, _left menu_ is usually fine.